content/src/roadmaps/license-tasks/landlord-registration.md

```diff
@@ -24,6 +27,14 @@ If you are renting out a residential property with 3 or more units, you need to
 
 Each town has its own application form and process, so check with your municipal clerk.
 
+${beginLocationDependentSection}
+
+**Check with your local clerk’s office**
+
+You can find your local clerk's office through the [${municipality} website](${municipalityWebsite}) or by contacting your [county clerk](${countyClerkWebsite}) at ${countyClerkPhone}.
```

Comment:
[dust] We have their county name, so should we have this be `[${countyName} county clerk]` instead?

Reply:
I'll bring this up to content for their thoughts